images/macos/macos-12-Readme.md

```diff
@@ -83,31 +83,32 @@
 - mongod 5.0.14
 - OpenSSL 1.1.1t  7 Feb 2023
 - Packer 1.8.6
+- pkg-config 0.29.2
 - PostgreSQL 14.7 (Homebrew)
 - psql (PostgreSQL) 14.7 (Homebrew)
 - Sox 14.4.2
 - Subversion (SVN) 1.14.2
 - Switchaudio-osx 1.1.0
 - Vagrant 2.3.4
 - VirtualBox 6.1.38r153438
-- yq 4.30.8
+- yq 4.31.1
 - zstd 1.5.4
 
 ### Tools
 - App Center CLI 2.13.2
-- AWS CLI 2.10.1
-- AWS SAM CLI 1.73.0
+- AWS CLI 2.10.2
+- AWS SAM CLI 1.74.0
 - AWS Session Manager CLI 1.2.398.0
 - Azure CLI 2.45.0
 - Azure CLI (azure-devops) 0.26.0
-- Bicep CLI 0.14.46
+- Bicep CLI 0.14.85
 - Cabal 3.6.2.0
 - Cmake 3.25.2
 - CodeQL Action Bundles 2.12.1 2.12.2
 - Colima 0.5.2
-- Fastlane 2.211.0
+- Fastlane 2.212.0
```

Comment:
You should really consider 2.212.1 since it includes a fix for an issue where Apple account gets locked.